### ISSUE TITLE
Is it possible to get exact time when first frame is captured?

### ISSUE THREAD

**Issue by pie6k** [wulkano/aperture-node#28]
Thank you for this amazing lib!

I record screen and mouse movement info at the same time. I later 'blend it' together, so I need to know the exact time when recording actually started.

`.startRecording()` promise resolves when recording starts, but it can be delayed by multiple reasons, eg main thread being busy. 

It would be great if `.startRecording()` would return promise with something like `firstFrameTimestamp`

```ts
const { firstFrameTimestamp } = await recorder.startRecording(options)
```

This way I would know exactly at which point in time actual visual recording starts.

Do you have any suggestions how can I do that (even if it requires some hacks)

**Comment by sindresorhus** [comment]
I agree, it would be useful to include the timestamp, however, it's unfortunately not something I have time to look into right now. PR welcome though. I think you could include the timestamp in one of the events sent from the Swift back-end.
